$OpenBSD: patch-include_efi_h,v 1.1 2019/04/09 22:32:48 patrick Exp $

Use 1ULL to make sure that when EFI_PAGE_MASK is used to mask
bits on 64-bit integers on 32-bit platforms it is 64-bit wide.

Index: include/efi.h
--- include/efi.h.orig
+++ include/efi.h
@@ -190,7 +190,7 @@ enum efi_mem_type {
 #define EFI_MEM_DESC_VERSION	1
 
 #define EFI_PAGE_SHIFT		12
-#define EFI_PAGE_SIZE		(1UL << EFI_PAGE_SHIFT)
+#define EFI_PAGE_SIZE		(1ULL << EFI_PAGE_SHIFT)
 #define EFI_PAGE_MASK		(EFI_PAGE_SIZE - 1)
 
 struct efi_mem_desc {
